plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveFileWriterFactory.java

```diff
@@ -36,5 +37,6 @@ Optional<FileWriter> createFileWriter(
             OptionalInt bucketNumber,
             AcidTransaction transaction,
             boolean useAcidSchema,
-            WriterKind writerKind);
+            WriterKind writerKind,
+            DateTimeZone parquetTimeZone);
```

Comment:
Rollback the changes on the interface, because as far as I understand, these changes affect only the `ParquetFileWriterFactory` and not all the `HiveFileWriterFactory` implementations.

I was a bit confused why do we have `UTC` passed as a constant in the Orc and Rc file writer factories.